src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj

```diff
@@ -3,9 +3,10 @@
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <RootNamespace>Microsoft.AspNetCore</RootNamespace>
-    <!-- https://github.com/dotnet/aspnetcore/issues/7939: This unit test requires the shared framework be available in Helix. -->
-    <BuildHelixPayload>false</BuildHelixPayload>
     <VerifyAncmBinary Condition="'$(TargetOsName)' == 'win' AND '$(TargetArchitecture)' != 'arm'">true</VerifyAncmBinary>
+    <TestDependsOnAspNetRuntime>true</TestDependsOnAspNetRuntime>
+    <TestDependsOnAspNetRef>true</TestDependsOnAspNetRef>
+    <SkipHelixArm>true</SkipHelixArm>
```

Comment:
To confirm: This is the _only_ change from the prior PR reneabling the SharedFx tests?

Reply:
Yep, I'll add the fix runtests fix and back out the extra queues